scripts/removeOldKibanaIndices.py

```diff
@@ -2,16 +2,15 @@
 
 logFile = open('/var/log/probe/KibanaStartup.log', 'w')
```

Comment:
Can you fix this too? I think it should be a `w+` so the write is appended to the startup log.